e2e/tests/localAuthorityMaintainsCaseAfterSubmission_test.js

```diff
@@ -0,0 +1,52 @@
+const config = require('../config.js');
+
+let caseId;
+
+Feature('Case maintenance after submission');
+
+Before(async (I, caseViewPage, submitApplicationEventPage) => {
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+    await I.enterMandatoryFields();
+    caseViewPage.goToNewActions(config.applicationActions.submitCase);
+    submitApplicationEventPage.giveConsent();
+    I.continueAndSubmit();
+
+    // eslint-disable-next-line require-atomic-updates
+    caseId = await I.grabTextFrom('.heading-h1');
+    console.log(`Case ${caseId} has been submitted`);
+  } else {
+    await I.navigateToCaseDetails(caseId);
+  }
+});
+
+Scenario('local authority uploads documents', (I, caseViewPage, uploadDocumentsEventPage) => {
+  caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+  uploadDocumentsEventPage.selectSocialWorkChronologyToFollow(config.testFile);
+  uploadDocumentsEventPage.uploadSocialWorkStatement(config.testFile);
+  uploadDocumentsEventPage.uploadSocialWorkAssessment(config.testFile);
+  uploadDocumentsEventPage.uploadCarePlan(config.testFile);
+  uploadDocumentsEventPage.uploadSWET(config.testFile);
+  uploadDocumentsEventPage.uploadThresholdDocument(config.testFile);
+  uploadDocumentsEventPage.uploadChecklistDocument(config.testFile);
+  uploadDocumentsEventPage.uploadAdditionalDocuments(config.testFile);
+  I.continueAndSave();
+  I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+  caseViewPage.selectTab(caseViewPage.tabs.documents);
+  I.seeDocument('Social work chronology', '', 'To follow', 'mock reason');
+  I.seeDocument('Social work statement and genogram', 'mockFile.txt', 'Attached');
+  I.seeDocument('Social work assessment', 'mockFile.txt', 'Attached');
+  I.seeDocument('Care plan', 'mockFile.txt', 'Attached');
+  I.seeDocument('Social work evidence template (SWET)', 'mockFile.txt', 'Attached');
+  I.seeDocument('Threshold document', 'mockFile.txt', 'Attached');
+  I.seeDocument('Checklist document', 'mockFile.txt', 'Attached');
+});
+
+Scenario('local authority uploads court bundle', (I, uploadDocumentsEventPage, submitApplicationEventPage, caseViewPage) => {
```

Comment:
Hmm, wasn't this blocked for local authority prior the submission? 
```
Scenario('Ability for a local authority to upload court bundle only after case is submitted', async (I, uploadDocumentsEventPage, submitApplicationEventPage, caseViewPage) => {
  I.dontSee('Court bundle');
  I.continueAndSave();
  I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);

```

Reply:
I found `I.dontSee('Court bundle')` assertion useless so removed it. Good spot though.

Reply:
That's right the case is already submitted so the test is ok